### DISCUSSION TITLE
NEW on Meshroom, can't get a good result

### DISCUSSION THREAD

**Discussion by topogigiojp** [alicevision/Meshroom#2634] [Q&A]
hello All
I'm new on meshroom, i take some pictures and they are all take in consideration, put i cannot get a form/geometries from the cameras
i tried a lot of approach but i get the same result.
the setting are all by default and i use my smartphone as camera.
screenshot in attachment.
If someone could help i appreciate.
Best regards
![Capture1](https://github.com/user-attachments/assets/ad332b86-0892-43d9-a770-86797dfda597)
![Capture2](https://github.com/user-attachments/assets/f27b5c62-4208-4635-8846-938798b911eb)

**Comment by natowi** [comment]
If this is your first time with Meshroom, choose something easier to reconstruct (your mug is featureless and reflective, both is difficult to reconstruct). Better choose something with natural texture like a rock or wood. I´d recommend to read up on capturing images for photogrammetry best practices.

**Comment by topogigiojp** [comment]
Hello Natowi.
Thank you a lot for the info. A will read the rest, i didn't yet this part.
I will try something else. The mug was closer lol. But thank you.
Best regards

A quarta, 1/01/2025, 13:39, natowi ***@***.***> escreveu:

**Reply by natowi** [reply]
https://github.com/alicevision/Meshroom/wiki/Tutorials
